src/main/resources/org/jenkinsci/plugins/workflow/support/visualization/table/FlowGraphTable/ajax.jelly

```diff
@@ -28,6 +28,9 @@
 -->
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
   <l:ajax>
+    <j:if test="${!ajax}">
```

Comment:
Really belongs in https://github.com/jenkinsci/workflow-job-plugin/blob/130687fcfd2eb546728dfdb1ac6666f5af174a91/src/main/resources/org/jenkinsci/plugins/workflow/job/views/FlowGraphTableAction/index.jelly#L30 but that would be awkward.